vividus/src/main/java/org/vividus/configuration/VaultStoredPropertiesProcessor.java

```diff
@@ -66,6 +69,17 @@ protected String processValue(String propertyName, String partOfPropertyValueToP
                 ));
     }
 
+    @Override
+    public String processProperty(String propertyName, String propertyValue)
+    {
+        if (propertyValue.matches("(?<!AZURE_KEY_|HASHI_CORP_)VAULT\\((.*?)\\)"))
```

Comment:
it should catch partial properties as well:
```
my-property=fooVAULT(secret-key)bar

Reply:
and it works for such cases also:
<img width="569" alt="Screenshot 2024-07-08 at 18 28 14" src="https://github.com/vividus-framework/vividus/assets/35694955/cf9b22c1-35ed-4ff4-ae0d-eb33725ae078">

Reply:
the screenshot is for "find", not for "matches"